src/SonarScanner.MSBuild/SonarScanner.MSBuild.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFrameworks>$(ScannerNetCoreVersion);$(ScannerNetFxVersion);$(ScannerNetVersion)</TargetFrameworks>
     <AssemblyName>SonarScanner.MSBuild</AssemblyName>
     <OutputType>Exe</OutputType>
+    <RollForward>LatestMajor</RollForward>
```

Comment:
I haven't thought about this.
I guess it's safer to be `Major` in this case, if it will be available. As it's more likely we would have tested it...

Reply:
I'd leave this for a separate PR as I'd need to do additional manual testing (as we don't have a regression functional test) and it's not very important at this point. 

I've opened https://github.com/SonarSource/sonar-scanner-msbuild/issues/1117 and we can decide if it's necessary for this release or if it can be done together with #1112